redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/openshift-pipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-1
```

Comment:
Shouldn't the image be `quay.io/openshift-pipeline/tektoncd-pipeline-operator`?

Reply:
updated